src/test/java/io/reactivex/internal/operators/flowable/FlowablePublishTest.java

```diff
@@ -1368,4 +1368,73 @@ public String apply(Integer t) throws Exception {
     public void badRequest() {
         TestHelper.assertBadRequestReported(Flowable.range(1, 5).publish());
     }
+
+    @Test
```

Comment:
nit: add link to original bug? https://github.com/ReactiveX/RxJava/issues/6363

Reply:
I don't usually include the issue number, that's what GitHub is for.